TVShows/Podfile

```diff
@@ -0,0 +1,21 @@
+# Uncomment the next line to define a global platform for your project
+platform :ios, '11.0'
+
+target 'TVShows' do
+  # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for TVShows
+  pod 'SVProgressHUD', '~> 2.2'
```

Comment:
Since you have used comments to group pods you are probably aware of issues with Podfile and grouping things. Comments are fine, but there is better way. Podfile uses Ruby syntax, so you can define helper method to group pods, something like this:

```ruby
def reactive
    pod 'RxSwift'
    pod 'RxCocoa'
end

def localization
    pod 'SwiftI18n/I18n+Case'
end

def core
    pod 'SwiftyBeaver'
    pod 'KeychainAccess'
end

def ui
    pod 'MBProgressHUD'
end

def networking
    pod 'Alamofire'
    pod 'AlamofireNetworkActivityIndicator'
    pod 'AlamofireNetworkActivityLogger'
    pod 'Loggie'
    pod 'CodableAlamofire'
end

def analytics
  pod 'Fabric'
  pod 'Crashlytics'
end

def testing
    pod 'RxTest'
    pod 'RxBlocking'
    pod 'Quick'
    pod 'Nimble'
    pod 'RxNimble'
end

def shared
    core
    ui
    networking
    analytics
end

target 'TVShows' do
   use_frameworks!
   shared
end
```